dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/loadbalance/AbstractLoadBalance.java

```diff
@@ -75,10 +75,16 @@ protected int getWeight(Invoker<?> invoker, Invocation invocation) {
         if (weight > 0) {
             long timestamp = invoker.getUrl().getParameter(REMOTE_TIMESTAMP_KEY, 0L);

```

Comment:
pls. change `REMOTE_TIMESTAMP_KEY` to `TIMESTAMP_KEY`. Pls. also remove org.apache.dubbo.rpc.cluster.Constants#REMOTE_TIMESTAMP_KEY since it's now useless.